pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/pancan/PancanOsteoRedirectUpdates.java

```diff
@@ -95,7 +105,186 @@ private interface SqlHelper extends SqlObject {
                 + "where workflow_transition_id in (<workflowTransitionIds>)")
         int deleteOsteoRedirectWorkflowTransitions(@BindList("workflowTransitionIds") List<Long> workflowTransitionIds);
 
-    }
 
+        @SqlUpdate("update expression\n" +
```

Comment:
copied the SQLs from MySQL workbench, hence not so clean formatting.
Didn't want to risk formatting and running into quotes mismatch kind of stuff.